src/components/ContactUs/ContactUsForm/ContactUsForm.yup.ts

```diff
@@ -9,7 +9,7 @@ export const validationSchema = Yup.object({
     )
     .required('validation.input.required.message'),
   country: Yup.string()
-    .max(30, 'contact.us.form.input.company.validation')
+    .min(1, 'contact.us.form.input.company.validation')
```

Comment:
is it correct that country has a validation of 1?

Reply:
Fixed